src/makeRobot.js

```diff
@@ -38,7 +38,64 @@
  * @return {object}
  */
 function makeRobot(name, wheels, version) {
-  // write code here
+  const robot = {
+    'name': name,
+    'wheels': wheels,
+    'version': version,
+    'coords': {
```

Comment:
If object properties not consist of several parts, it may be without quotes.

Reply:
Yes, if keys are valid variable names, you don't need `''`.
Also, if `variableName === keyName` you can use short syntax:
```suggestion
    name,
    wheels,
    version,
    coords: {
```